### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by DChan0319**

### CODE REVIEW

File: client/src/components/dashPage/clientTab.jsx [hunk 1]
```diff
@@ -48,6 +48,7 @@ class ClientTab extends Component {
 
   renderCards(cardList) {
     let content = [];
+    console.log("this is cardlist", cardList);
```
Comment (zgallagher08) [inline client/src/components/dashPage/clientTab.jsx]: remove console log

File: client/src/components/dashPage/clientTab.jsx [hunk 2]
```diff
@@ -110,11 +112,49 @@ class ClientTab extends Component {
...
+    </div>}
+  </div>
+</div>   
+    );
   }
 }
   render() {
     return (
       <div>
+        {console.log(this.state.cardList)}
```
Comment (zgallagher08) [inline client/src/components/dashPage/clientTab.jsx]: remove console log

**Review by zgallagher08 — CHANGES_REQUESTED** [review]
looks good, just remove logs

**Review by zgallagher08 — APPROVED** [review]

File: client/src/components/dashPage/searchItem.jsx [hunk 3]
```diff
@@ -23,9 +23,6 @@ class SearchItem extends Component {
     return(
       <div className="dropdown-item">
         {
-          this.props.picture ?
```
Comment (zgallagher08) [inline client/src/components/dashPage/searchItem.jsx:26]: did you intend to delete this?

**Review by amelendez37 — APPROVED** [review]

**Review by jaba9531 — APPROVED** [review]
Thanks, great work

**Pull request merged**